webapp/src/css/enketo/medic.less

```diff
@@ -445,6 +445,12 @@
   }
 
   #duplicate_contacts {
+    display: none;
```

Comment:
I think we only want to show the duplicates panel _on the last page_ of the form (right before the "Submit" button).  If a user tries to submit a form and there are dupes found, then they use the "Previous" button to go back to an earlier page in the form, they should not still see the duplicates panel.

The easiest way I could come up with to only show the duplicates on the last page was to put it into the `.form-footer` and then control the display here based on if the `.form-footer` had the `.end` class.  All the logic for adding the `.end` class to the `.form-footer` is already existing (and controls the "Submit" vs "Next" button).